packages/fxa-payments-server/src/lib/test-utils.tsx

```diff
@@ -9,6 +9,9 @@ import {
 import ScreenInfo from '../../src/lib/screen-info';
 import { ReactStripeElements } from 'react-stripe-elements';
 import nock from 'nock';
+import fs from 'fs';
+import path from 'path';
+import { FluentBundle } from 'fluent';
```

Comment:
@jackiemunroe I see that you haven't upgraded to `@fluent/bundle` 0.15 nor `@fluent/react` 0.11 in this PR. Did you run into any trouble? I might be able to help :) Or, is it better to upgrade in a separate PR?

Reply:
@stasm Since there was a decent amount of changes here, I thought it would be better to upgrade fluent in isolation.